undertow/src/main/resources/org/wildfly/extension/undertow/LocalDescriptions.properties

```diff
@@ -234,6 +234,8 @@ undertow.listener.tcp-backlog=Configure a server with the specified backlog.
 undertow.listener.tcp-keep-alive=Configure a channel to send TCP keep-alive messages in an implementation-dependent manner.
 undertow.listener.read-timeout=Configure a read timeout for a socket, in milliseconds.  If the given amount of time elapses without a successful read taking place, the socket's next read will throw a {@link ReadTimeoutException}.
 undertow.listener.write-timeout=Configure a write timeout for a socket, in milliseconds.  If the given amount of time elapses without a successful write taking place, the socket's next write will throw a {@link WriteTimeoutException}.
+undertow.listener.websocket-read-timeout=Configure a read timeout for a web-socket, in milliseconds. If the given amount of time elapses without a successful read taking place, the socket's next read will throw a {@link ReadTimeoutException}. If specified, it overrides general read-timeout.
```

Comment:
What is the default when unspecified? This information is missing

Reply:
"If specified, it overrides general read-timeout." :)